mindsdb/integrations/handlers/pgvector_handler/README.md

```diff
@@ -62,19 +62,51 @@ WITH
 You can insert data into a new collection like so
 
 ```sql
-CREATE TABLE pvec.items
-    (SELECT embedding AS embeddings
-        FROM (SELECT * FROM mysql_demo_db.demo_fda_context
-        LIMIT 3) AS d
-join  openai_emb);
+--- Public demo database
+CREATE DATABASE mysql_demo_db
+WITH ENGINE = "mysql",
+PARAMETERS = {
+   "user": "user",
+   "password": "MindsDBUser123!",
```

Comment:
Let's change the values of host, user, and password to some placeholders.

Reply:
@martyna-mindsdb, I updated the README to remove the public demo database. Is it ok?